frontend/src/app/pages/integrations/IntegrationDialog.tsx

```diff
@@ -124,6 +125,15 @@ export const IntegrationDialog = (props: IntegrationDialogProps) => {
                     propertyType = propertyType.max(property.maxLength, texts.validation.maxLengthFn);
                 }
 
+                if (property.format && property.format !== "None") {
+                    const format = FORMAT_REGEXPS.get(property.format);
+
+                    
+                    if (format) {  
+                        propertyType = propertyType.matches(format, texts.validation.formatFn);
```

Comment:
Yup already provides validation method for email and url. Just use that. There is also emailI18n and perhaps url18n for localized methods in the code base.

Reply:
It makes perfectly sense to support localhost :)